scripts/generate-questions-csv.py

```diff
@@ -12,12 +12,15 @@
 pip install -r requirements.txt
 
 Usage:
-    scripts/generate-questions-csv.py <DM-content-root-dir> <output-directory> --framework=<slug> [--context=<yaml>]
+    scripts/generate-questions-csv.py <DM-content-root-dir> <output-file> [options]
 
+--framework=<slug>
+--question-set=<question-set-directory-name>
+--context=<yaml>
 -h --help    show this message
 
 Example:
-    scripts/generate-questions-csv.py /path/to/dm-frameworks/ ~ --framework=g-cloud-9 --context="lot: SaaS"
+    scripts/generate-questions-csv.py /path/to/dm-frameworks/ ~/mystuff.csv --framework=g-cloud-9 --context="lot: SaaS"
```

Comment:
Ah, OK, I see, we can only exclude things that have a `depends: on: lot...` entry in the content using the context.